tools/rosunit/src/rosunit/pyunit.py

```diff
@@ -84,7 +86,12 @@ def unitrun(package, test_name, test, sysargs=None, coverage_packages=None):
         start_coverage(coverage_packages)
 
     # create and run unittest suite with our xmllrunner wrapper
-    suite = unittest.TestLoader().loadTestsFromTestCase(test)
+    suite = None
+    if issubclass(test, unittest.TestCase):
+        suite = unittest.TestLoader().loadTestsFromTestCase(test)
+    else:
+        suite = unittest.TestLoader().loadTestsFromName(test)
```

Comment:
This fails. I guess it broke when simplifying the logic in #101 when merging in PR #104. 

The intention is for a string to pass through the first check into the `loadTestsFromName` call, but it never gets there since the first method throws a `TypeError`.

```
$ python
Python 2.7.6 (default, Mar 22 2014, 22:59:56) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import unittest
>>> issubclass("foo", unittest.TestCase)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: issubclass() arg 1 must be a class
>>>
```